src/Evolution/VariableFixing/RadiallyFallingFloor.cpp

```diff
@@ -57,6 +60,60 @@ void RadiallyFallingFloor<Dim>::operator()(
     density->get()[i] =
         std::max(density->get()[i], rest_mass_density_scale_ *
                                         pow(radius, rest_mass_density_power_));
+    if constexpr (ThermodynamicDim == 1) {
+      // For the 1D EoS, density will be used to calculate the remaining
+      // primitives, even overwriting pressure.  This decision prioritizes
+      // thermodynamic consistency over the user defined pressure profile.
+      pressure->get()[i] = get(equation_of_state.pressure_from_density(
+          Scalar<double>{density->get()[i]}));
+
+      specific_internal_energy->get()[i] =
+          get(equation_of_state.specific_internal_energy_from_density(
+              Scalar<double>{density->get()[i]}));
+
```

Comment:
Should this also recompute `T`? Since that's what we reconstruct that needs to be consistent

Reply:
Updated.  I thought temperature defaults to 0 for 1D EoS, but I have included it.